ibl-frontend/frontend-content/src/app/overview/overview.component.html

```diff
@@ -42,39 +42,50 @@ <h2 class="top-title">International Brain Laboratory Behavioral Data Portal</h2>
 
   <div class="home-shaded-container">
     <div class="detail-container">
-      <h3>How can you access the data?</h3>
-      <ol class="detail-text">
-        <li>To access the <b>data associated with the electrophysiology recordings</b>, follow <a href="https://int-brain-lab.github.io/iblenv/public_docs/public_one.html" target="_blank">
-          these guidelines</a>.
-        </li>
-        <li>To access the <b>behavior data</b> used in this online portal, you have three options:
```

Comment:
I see that you removed these. Does that mean that *data associated with the electrophysiology recordings* + *behavior data* are now both compatible with all the options?